### PR TITLE
First draft of gallina-extensions.rst

### PR DESCRIPTION
**Pull request by psteckler**
Also have draft of MIGRATING document for others to follow

Note: the .rst file is based on the HTML file for 8.7, not on the refman in the sphinx-doc branch, which is out-of-date.

### CODE REVIEW

File: doc/sphinx/MIGRATING [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+ (global-set-key (kbd "<f12>") #'sphinx/rst-coq-action)
+
+ With this code installed, you can hit "F12" followed by an appropriate key to do quick markup of text.
+
+# Fork Maxime's repo:
+
+ https://github.com/maximedenes/coq
+ 
+ N.B.: if you already have a Github Coq repo, you'll need to create a second GitHub account, and fork it to that account
```
Comment (maximedenes) [inline doc/sphinx/MIGRATING]: I don't think that's required.

**Comment by maximedenes** [comment]
I see a few warnings:

```
WARNING: Line block ends without a blank line.
WARNING: Enumerated list ends without a blank line; unexpected unindent.
WARNING: Explicit markup ends without a blank line; unexpected unindent.
WARNING: Undefined substitution referenced: "module_type_1` <+ … <+ |module_type_n".
```

**Comment by psteckler** [comment]
@maximedenes I made these fixes.

The line-block text is now in a pale green, via CSS magic. That can be changed, of course.

**Comment by psteckler** [comment]
Note that travis does not have `sphinx-build` installed, so doc builds fail there.

**Comment by psteckler** [comment]
Clément had some nice formatting in his examples that I didn't use in my draft. I'll add those.

So don't merge until that's done.

**Comment by maximedenes** [comment]
> So don't merge until that's done.

Ok, let me know when ready to merge.

**Comment by maximedenes** [comment]
> Note that travis does not have sphinx-build installed, so doc builds fail there.

Yes, I'll try to fix that.

**Comment by psteckler** [comment]
I've made the changes I mentioned, squashed the commits. For me, the build has no warnings except about the undefined cross-references.

See if the output looks OK, and merge if acceptable.

**Pull request merged**